src/utils.ts

```diff
@@ -41,6 +41,13 @@ export function isSPDXValid(license: string): boolean {
   }
 }
 
+function isEnterprise(): boolean {
+  const serverUrl = new URL(
+    process.env['GITHUB_SERVER_URL'] ?? 'https://github.com'
```

Comment:
GITHUB_SERVER_URL is [always set](https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables) in the Action Environment Variables but process.env returns string | undefined